vars/alvariumGetAnnotators.groovy

```diff
@@ -58,17 +45,9 @@ def call(List<String> annotatorKinds, String artifactPath=null) {
                     artifactPath,
                     checksum.getAbsolutePath()
                 )
```

Comment:
Is the File object needed?
Can we pass the path directly as the source-code case?